units/systemd-binfmt.path

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Path activation for additional binary formats
+Documentation=man:systemd-binfmt.service(8) man:binfmt.d(5)
+Documentation=https://www.kernel.org/doc/Documentation/binfmt_misc.txt
+
+[Path]
+DirectoryNotEmpty=/lib/binfmt.d
+DirectoryNotEmpty=/usr/lib/binfmt.d
+DirectoryNotEmpty=/usr/local/lib/binfmt.d
```

Comment:
I find these directories are actually used, and the man pages of other similar utils do not mention these directories. So, please forget my previous comments.